common/src/main/java/com/microsoft/identity/common/internal/controllers/LocalMSALController.java

```diff
@@ -213,7 +214,7 @@ public AcquireTokenResult acquireToken(
     private AuthorizationResult performAuthorizationRequest(@NonNull final OAuth2Strategy strategy,
                                                             @NonNull final Context context,
                                                             @NonNull final InteractiveTokenCommandParameters parameters)
-            throws ExecutionException, InterruptedException, ClientException {
+            throws ExecutionException, InterruptedException, ClientException, UnsupportedEncodingException {
```

Comment:
How are we ending up with UnsupportedEncoding here?

Reply:
It looks like it's declared as thrown from requestAuthorization in both implementations, but is not actually thrown there.